pxr/usdImaging/usdSkelImaging/extComputations.cpp

```diff
@@ -141,8 +141,9 @@ UsdSkelImagingInvokeExtComputation(
         return;
     }
 
-    VtVec3fArray skinnedPoints =
-        restPointsValue.UncheckedGet<VtVec3fArray>();
+    // Explicit copy of the skinnedPoints to avoid VT_LOG_STACK_ON_ARRAY_DETACH_COPY logs
+    VtVec3fArray skinnedPoints(restPointsValue.UncheckedGet<VtVec3fArray>().begin(), // TODO: MakeUnique()
```

Comment:
Thanks for adding the comment -- sorry if I wasn't clear in the earlier comments, but I'd like for us _not_ to add explicit copies, since if the data is already unique, this is a pessimization.

Reply:
Sorry @gitamohr , I didn't see this very comment. I'll do that.